.gitignore

```diff
@@ -5,7 +5,8 @@
 .terraform
 terraform.tfvars
 remote-state.tf
+remote_state.tf
```

Comment:
Why are adding this line a second time? Same for `eks/.gitignore`

Reply:
> gitignore already had remote-state.tf, but I tend to avoid dashes in filenames, which explains why I added remote_state.tf as well.

happy to either remove remote-state.tf from gitignore or change the readme to match remote-state.tf

Reply:
We are using `remote-state.tf` for the GKE part of the code (and we used to have a template file for EKS, I'm sure). For consistency, please update the Readme to use `remote-state.tf` and leave `remote_state.tf` out of `.gitignore` if that's ok

Reply:
updated ... got rid of the entry in .gitignore ... no need to duplicate